tsnet/tsnet.go

```diff
@@ -930,6 +930,9 @@ func (s *Server) ListenFunnel(network, addr string, opts ...FunnelOption) (net.L
 	if err != nil {
 		return nil, err
 	}
+	// TODO(sonia,tailscale/corp#10577): We may want to use the interactive enable
+	// flow here instead of CheckFunnelAccess to allow the user to turn on funnel
+	// if not already on. See cli.serveEnv.verifyFunnelEnabled.
```

Comment:
Only if we're super sure they're running it from a terminal. It's better to either continue on in degraded more or crash hard if they're running it under systemd/etc.

Reply:
Yea figured tsnet is probably a different case. I'll add a little more to this comment.